community-roles.md

```diff
@@ -46,107 +39,64 @@ This can be through code, documentation, taking part in bug scrubs, etc.
-
-**Requirements:**
-
-- Make a PR against the `MAINTAINERS` file for a `fluxcd` GitHub org repo. [Example PR](https://github.com/fluxcd/source-controller/pull/584)
-- @mention all the other current maintainers
-- Have maintainers submit their vote by `+1`
-- Once all maintainers in repo have `+1` the pr will be reviewed by a member of the Flux [Oversight Committee]
+Process: refer to [PROCESS.md](PROCESS.md#applying-for-flux-membership).
 
-**flux2 maintainers:** Maintainership in the [flux2 MAINTAINERS file](https://github.com/fluxcd/flux2/blob/main/MAINTAINERS) trickles down to all other Flux 2-related repositories which means that mantainers mentioned there are also maintainers in all other repositories.
+### Maintainer
 
-Once the above process has taken its course, make sure you
+Maintainers are elected [Project Members][Project Member] who have shown significant and sustained contributions in a Git repository.
 
-- Are added to the internal `#flux-maintainers` Slack channel
-- Update <https://maintainers.cncf.io>
-- Get somebody to ping [CNCF Service Desk](https://cncfservicedesk.atlassian.net/) to get you added as Flux maintainer
-- Ping fellow maintainers to get added to Flux 1Password
-- (Optional) Check if your [CNCF affiliation is up to date](https://github.com/cncf/gitdm#addingupdating-affiliation)
+**Defined by:** entry in MAINTAINERS file in a repo owned by the Flux project, and membership in the [@fluxcd/maintainers](https://github.com/orgs/fluxcd/teams/maintainers) GitHub team.
 
 **Responsibilities and Privileges:**
 
-The following apply to the part of the codebase for which one would be in a MAINTAINERS file:
+To become a Maintainer you need to demonstrate the following:
 
 - Enable and promote Flux community values
 - Engage with end Users through appropriate communication channels
-- Serve as a point of conflict resolution between Contributors to their git repository
+- Serve as a point of conflict resolution between Contributors to their Git repository
 - Maintain open collaboration with Contributors and other Maintainers
 - Ask for help when unsure and step down considerately
+- A good understanding of the code-base (or equivalent that is governed by the repository, e.g. `fluxcd/community` or `fluxcd/website`)
+- Willing to take on long-term responsibility for the project (or a specific part of it)
+- Commitment to the project. Specifically:
 
-### Oversight Committee
+  This can be evidenced differently and we want to maintain some general flexibility assessing this. Significant and sustained contributions can be be both by showing a long-term level of care with a bigger number of smaller contributions or by a smaller set of sizable contributions. To make it somewhat more comparable, here is an example of commitment we would be happy to accept for a maintainer:
```

Comment:
```suggestion
  This can be evidenced differently and we want to maintain some general flexibility assessing this. Significant and sustained contributions can be both by showing a long-term level of care with a bigger number of smaller contributions or by a smaller set of sizable contributions. To make it somewhat more comparable, here is an example of commitment we would be happy to accept for a maintainer:
```